index.html

```diff
@@ -1,16 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Cake webpage</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+  <body>
+    <main class="homepage-main">
+      <section class="homepage-hero">
+        <div class="one-one">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+        <div class="one-two">The best cake in town <br>delivered to your door</div>
+        <div class="navigation">
```

Comment:
I would say, looking at the design that these belong to a second block without the background image